bootstrap/main.py

```diff
@@ -4,15 +4,19 @@
 import sys
 
 import docker
-
+from loguru import logger
 from bootstrap.bootstrap import Bootstrapper
 
+
 if __name__ == "__main__":
+    logger.add("/var/logs/blueos/services/bootstrap/bootstrap.log", level="DEBUG", rotation="1 MB")
```

Comment:
Will this work if BlueOS rotator runs ?

Reply:
I have no idea. should I just take this out? I guess the other services don't any rotation by themselves, right?

Reply:
Let it run and check if there is a rotation and it's working fine, right now we have a default rotation of 10min:
https://github.com/bluerobotics/BlueOS/blob/master/core/libs/commonwealth/commonwealth/utils/logs.py#L63

And a zipper every hour:
https://github.com/bluerobotics/BlueOS/blob/master/core/start-blueos-core#L78